src/main/scala/chisel3/util/random/PRNG.scala

```diff
@@ -33,15 +33,19 @@ abstract class PRNG(val width: Int, val seed: Option[BigInt], step: Int = 1, upd
 
   val io: PRNGIO = IO(new PRNGIO(width))
 
+  /** Allow implementations to override the reset value, e.g., if some bits
+    * should be don't-cares. */
```

Comment:
Mega-nit... Scaladoc guidelines on multi-line comments are to add an extra `*/`. Alternatively, I think this is short enough to fit on one line and stay under the 120 char limit:
```suggestion
    * should be don't-cares. 
    */
```

Reply:
Fixed in rebase.